src/lib/draw-and-pick.js

```diff
@@ -354,47 +354,47 @@ function getPickedColors(gl, {
   // Make sure we clear scissor test and fbo bindings in case of exceptions
   // We are only interested in one pixel, no need to render anything else
   // Note that the callback here is called synchronously.
-  return glContextWithState(gl, {
+  return withParameters(gl, {
     frameBuffer: pickingFBO,
     framebuffer: pickingFBO,
     scissorTest: {x, y, w: width, h: height}
```

Comment:
Oh yeah scissorTest and scissor, this is the old code, you already fixed it :), will reflect when I update the PR.